e2e-tests/src/test/rewards.rs

```diff
@@ -311,3 +304,82 @@ pub fn force_new_era(config: &Config) -> anyhow::Result<()> {
 
     Ok(())
 }
+
+pub fn change_stake_and_force_new_era(config: &Config) -> anyhow::Result<()> {
```

Comment:
maybe it is out of scope of this PR, but I feel that our E2E tests should be documented, i.e. every test function should have a docstring describing the scenario and the expected outcome

if you agree, you can add such description here and then we will create Jira with your test as the example